polars/polars-core/src/frame/mod.rs

```diff
@@ -240,15 +240,19 @@ impl DataFrame {
             let mut names = PlHashSet::with_capacity(series_cols.len());
 
             for s in &series_cols {
+                let name = s.name();
```

Comment:
I think we should get the first name only during raising an error. The happy path should not be slowed down by giving extra error information.

Reply:
You're right. Changed it around, should be good to go now!